web/ui/react-app/src/App.css

```diff
@@ -212,6 +212,26 @@ div.time-input {
     margin-bottom: 20px;
 }
 
+.target-head {
+    font-weight: 700;
+    font-size: large;
+}
+
+.label-component {
+    margin-bottom: 1%;
+}
+
+.table-outer-layer {
+    margin: 1%;
+    margin-right: 5%;
+}
+
+.head {
```

Comment:
I don't think the `head` class is used anywhere? But probably these CSS comments will be obsolete anyway when making the structure / styling more consistent with the targets page.